LICENSE.txt

```diff
@@ -1,88 +0,0 @@
-Copyright 2015-2019 Skymind Inc.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-
-
-##########################
-
--- Misc files --
```

Comment:
For all of the files that can't have a copyright header directly, we will need to have an entry here. So we'll need both to keep this file and update it. I think it also needs an updated copyright header.

Reply:
I've updated the LICENSE.txt file. All the files with code was edited, sometimes just package name changes. Shall add the Konduit KK copyright to all the files then? Thanks.

Reply:
Added Konduit copyright